### DISCUSSION TITLE
%E5%85%A5%E5%AD%A6%E7%AF%87/%E5%BC%80%E5%AD%A6%E6%90%BA%E5%B8%A6%E7%9A%84%E4%B8%9C%E8%A5%BF

### DISCUSSION THREAD

**Discussion by giscus** [SurviveXJTU/SurviveXJTU.github.io#11] [Announcements]
# %E5%85%A5%E5%AD%A6%E7%AF%87/%E5%BC%80%E5%AD%A6%E6%90%BA%E5%B8%A6%E7%9A%84%E4%B8%9C%E8%A5%BF

西安交大生存指南，更适合西交大学子的生存指南。

https://survivexjtu.github.io/%E5%85%A5%E5%AD%A6%E7%AF%87/%E5%BC%80%E5%AD%A6%E6%90%BA%E5%B8%A6%E7%9A%84%E4%B8%9C%E8%A5%BF.html

**Comment by SmartPig-Joe** [comment]
需要蚊帐吗，如果需要，尺寸需要多少的

**Reply by Axi404** [reply]
蚊帐是可供选择的，同时床帘可以提供隐私空间，一般来说蚊帐选购均是直接根据床铺尺寸，不同宿舍的高度则又不相同，可能需要到校后具体测量（同专业的学长大概率和你不在一个宿舍）。宿舍的夏天会存在蚊虫（假如没有谨慎关门关窗），但是有必要指出的是，部分的床铺因为装修问题，在顶部存在横杆，可能为安装蚊帐/窗帘增加额外的困难。综上，其实建议到校后自己实际测量再进行选购。
